### PR TITLE
[FEATURE] Add Sign-in and Sign-up page

### PR DESCRIPTION
**Pull request by Nexucis**
# Description

This PR is adding a Sign-up and Sign-in page to be able to register new user and to authenticate themself.

Note: Token is not yet managed. I'm struggling a bit about how to manage it.

# Screenshots

https://github.com/perses/perses/assets/4548045/6ef6f026-e64e-478c-bde8-d83e0df95322

### CODE REVIEW

File: ui/app/src/views/auth/SignWrapper.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import LightThemePersesLogo from '../../components/logo/LightThemePersesLogo';
+
+export function SignWrapper(props: { children: React.ReactNode }) {
+  const { isDarkModeEnabled } = useDarkMode();
+  const isLaptopSize = useMediaQuery(useTheme().breakpoints.up('sm'));
+  return (
+    <Container
+      maxWidth="sm"
+      sx={{
+        display: 'flex',
```
Comment (Gladorme) [inline ui/app/src/views/auth/SignWrapper.tsx:28]: <Stack> is more adapted flex displays

File: ui/app/src/views/auth/SignUpView.tsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    <SignWrapper>
+      <Container
+        disableGutters={useMediaQuery(useTheme().breakpoints.up('sm'))}
+        sx={{
+          display: 'flex',
+          flexDirection: 'row',
+          justifyContent: 'space-between',
+          marginTop: '10px',
+          paddingLeft: 0,
+          paddingRight: 0,
```
Comment (Gladorme) [inline ui/app/src/views/auth/SignUpView.tsx:60]: From the video, I think we should add a `gap: 2` between these fields

File: ui/app/src/views/auth/SignWrapper.tsx (hunk 1 above)
Comment (Nexucis) [reply ui/app/src/views/auth/SignWrapper.tsx:28]: I tried

```jsx
    <Stack
      direction={{ xs: 'column', sm: 'row' }}
      justifyContent="center"
      alignItems="center"
      maxWidth="sm"
      divider={<Divider orientation={isLaptopSize ? 'vertical' : 'horizontal'} flexItem />}
      spacing={2}
    >
```

But it doesn't seem the `justifyContent` is working it positions the element on the left like if `justifyContent` was equal to `flex-start` :(

**Review by Gladorme — APPROVED** [review]
LGTM, I will probably do some testing on my side for Stack stuff

**Pull request merged**